device/cluster.cpp

```diff
@@ -678,6 +679,17 @@ void Cluster::configure_active_ethernet_cores_for_mmio_device(
     non_mmio_transfer_cores_customized = true;
 }
 
+void Cluster::configure_active_ethernet_cores_for_mmio_device(
```

Comment:
how are active vs. idle eth being derived in UMD today? I am putting together a document that will describe how this is being done for BH.

Reply:
Isn't `create-ethernet-map` returning list of connections between ETH cores across devices?

Reply:
It is, but this was just adding the same API that is public in Cluster to be the same just with CoreCoord. If this API ends up being not used, we will remove it. As far as idle vs active, since we have a meeting tomorrow for it, we will discuss it there